src/services/api/clinical-trials-search-api/__tests__/getClinicalTrials.test.js

```diff
@@ -0,0 +1,107 @@
+import { getClinicalTrials } from '../getClinicalTrials';
+import { getClinicalTrials as getClinicalTrialsQuery } from '../../actions/getClinicalTrials';
+import clinicalTrialsSearchClientFactory from '../clinicalTrialsSearchClientFactory';
+import axios from 'axios';
+import nock from 'nock';
+
+// Required for unit tests to not have CORS issues
+axios.defaults.adapter = require('axios/lib/adapters/http');
+/*
+const client = axios.create({
+	baseURL: 'https://clinicaltrialsapi.cancer.gov/v1',
+	timeout: 15000,
+});
+*/
+/*
+const nockClient = axios.create({
+	baseURL: 'https://example.org',
+	timeout: 15000,
+});
+*/
+const client = clinicalTrialsSearchClientFactory(
+	'https://clinicaltrialsapi.cancer.gov/v1'
+);
+const nockClient = clinicalTrialsSearchClientFactory('https://example.org');
+
+describe('testing getClinicalTrials', () => {
+	beforeEach(() => {
+		jest.clearAllMocks();
+	});
+
+	afterAll(() => {
+		nock.cleanAll();
+		nock.enableNetConnect();
+	});
+
+	test('makes a request to api and returns 57 trials, for trastuzumab', async () => {
+		const requestFilters = {
+			'arms.interventions.intervention_code': ['C1647'],
+		};
+
+		const query = getClinicalTrialsQuery({
+			requestFilters,
+		});
+		const response = await getClinicalTrials(client, query.payload);
+		expect(response.total).toEqual(57);
+	});
+
+	test('makes a request where the response is 404', async () => {
+		const badClient = axios.create({
```

Comment:
This does not seem to have a scope defined for nock. do it may be connecting to the internet.